Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/Merchant.yaml

```
@@ -826,6 +826,7 @@ TransporterConfig:
     thresholdCancellationScore: Maybe Int
     minRidesForCancellationScore: Maybe Int
     thresholdCancellationPercentageToUnlist: Maybe Int
+    qarCalRadiusInKm : Maybe Double
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)